ios/versioned-react-native/ABI49_0_0/ReactNative/React/ABI49_0_0React-RCTFabric.podspec

```
@@ -12,7 +12,7 @@ version = package['version']
 
 folly_flags = '-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1'
```

```suggestion
folly_flags = '-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1 -DFOLLY_CFG_NO_COROUTINES=1'
```

sorry i missed this one

---

Updated